package.json

```diff
@@ -1,9 +1,13 @@
 {
   "name": "awesome-electronics",
   "scripts": {
-    "toc": "markdown-toc README.md -i"
+    "toc": "markdown-toc README.md -i --bullets=-",
+    "lint": "awesome-lint"
   },
   "dependencies": {
     "markdown-toc": "^1.1.0"
+  },
+  "devDependencies": {
+    "awesome-lint": "^0.12.1"
```

Comment:
We should just put both of these either in dependencies or dev-. I don't mind which since it's not really a web project and the distinction is meaningless, but not point in seperating them.

Reply:
fixed, both moved to devDependencies